### PR TITLE
add genesis for protocol version 2

### PR DESCRIPTION
**Pull request by boundless-forest**
TODO: 
- tag and release
- update cita

### CODE REVIEW

File: genesis.json [hunk 1]
```diff
@@ -734,7 +734,7 @@
                 "0x11": "0xffffffffffffffffffffffffffffffffff021014",
                 "0x21": "0xffffffffffffffffffffffffffffffffff021025",
                 "0x1b": "0xffffffffffffffffffffffffffffffffff021001",
-                "0x25": "0xffffffffffffffffffffffffffffffffff02000c00000001",
+                "0x25": "0xffffffffffffffffffffffffffffffffff02000c00000002",
```
Comment (kaikai1024) [inline genesis.json:737]: Here is default protocal version: 2

**Review by kaikai1024 — APPROVED** [review]

File: genesis.json (hunk 1 above)
Comment (boundless-forest) [reply genesis.json:737]: 对啊, 这里不应该是这样吗?

Comment (kaikai1024) [reply genesis.json:737]: I just point it out, it's friendly for reviewers.

Comment (boundless-forest) [reply genesis.json:737]: OK :smiley:

**Pull request merged**